finance_manager/src/main/resources/db/changelog/changeset/insert-users.sql

```diff
@@ -0,0 +1,8 @@
+--liquibase formatted sql
+--changeset "gs":3
+
+INSERT INTO users (u_email, u_password, u_is_admin)

```

Comment:
Here we also need the precondition that user with such email does not exist in database

Reply:
Should I add UNIQUE contraint to "u_email" in users table too?

Reply:
Sure

Reply:
I tried to solve it using ON CONFLICT .... DO NOTHING and have issues  with H2 during tests because of compability problems with PostgreSQL. Maybe becuse initial users will be inserted using migrations (no registration function) using "ON CONFLICT .... DO NOTHING" during inserting users is not neccesary. Constraint UNIQUE in "email" column will prevent us from duplicating emails.

Reply:
Done